extensions/pkg/webhook/shoot/webhook.go

```diff
@@ -114,7 +120,18 @@ func ReconcileWebhooksForAllNamespaces(
 				return err
 			}
 
-			return ReconcileWebhookConfig(ctx, c, namespaceName, extensionNamespace, extensionName, managedResourceName, *shootWebhookConfigs.DeepCopy(), cluster)
+			if err := ReconcileWebhookConfig(ctx, c, namespaceName, extensionNamespace, extensionName, managedResourceName, *shootWebhookConfigs.DeepCopy(), cluster, false); err != nil {
+				statusErr := &apierrors.StatusError{}
+				if !errors.As(err, &statusErr) {
+					return err
+				}
+				// Ignore not found errors since the managed resource can be deleted in parallel during shoot deletion.
+				if apierrors.IsNotFound(err) {
+					return nil
+				}
```

Comment:
I'm confused 😄 Can't we get rid of L124-L127? Then we could also go back and use

```golang
if err := ReconcileWebhookConfig(ctx, c, namespaceName, extensionNamespace, extensionName, managedResourceName, *shootWebhookConfigs.DeepCopy(), cluster, false); client.IgnoreNotFound(err) != nil {
	return err
}
```

Reply:
Nice, thanks for the hint. The NotFound error is **wrapped** but this is supported by `apierrors.IsNotFound`, which I didn't know 🙂